apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -209,36 +224,21 @@ end_per_suite(_Config) ->
 
 init_per_group(all_endpoints, Config) ->
     Config;
-init_per_group(Group, Config) when Group =:= external_endpoints
-                            orelse Group =:= internal_endpoints ->
+init_per_group(_Group, Config) ->
```

Comment:
I expect that this change, because of the group-nesting group `all_endpoints`, makes this function body to be run twice (without the corresponding `end_per_group` having been run already). So the requirement on this function body is that it is idempotent. It is idempotent at the moment, as the node start can be run again and it leaves the node running, and and the aehttp app_started event is recorded in the proxy. Though for avoiding future breakages I suggest to either:
A. Restore start of node only for specified leaf groups (external_endpoints, internal_endpoints, websockets); or
B. Add comment that body here needs to be idempotent.

I recommend A.

Reply:
> A. Restore start of node only for specified leaf groups (external_endpoints, internal_endpoints, websockets); or

I had not noticed that clause before already handles `all_endpoints`, so please ignore me on this. Thanks for pointing that out.